assets/benchmarks/shaders/VsOutput.hlsli

```diff
@@ -15,9 +15,23 @@
 #ifndef BENCHMARKS_VS_OUTPUT_HLSLI
 #define BENCHMARKS_VS_OUTPUT_HLSLI
 
-struct VSOutputPos
-{
-    float4 position : SV_POSITION;
+struct ColorParams {
+  float3 Value;
+};
+
+struct ConfigParams {
```

Comment:
out of curiosity, would the shader compiler add padding in between `RandomSeed` and `Color`?

Reply:
ah, good to know, thanks!